DESCRIPTION

```diff
@@ -90,15 +90,20 @@ Imports:
     tools,
     utils,
     uuid (>= 0.1-4),
-    xml2 (>= 1.3.0)
+    xml2 (>= 1.3.0),
 Suggests:
     knitr,
     mime (>= 0.9),
     reader (>= 1.0.6),
     ritis (>= 1.0.0),
     taxize (<= 0.9.99),
     testthat,
-    worrms (>= 0.4.2)
+    worrms (>= 0.4.2),
+    scales,
+    ggrepel,
+    usmap,
+    rgdal,
```

Comment:
I don't see any functions referencing this package. Remove?

Reply:
I can't tell which package you're referring to here, but assuming rgdal, that one is required by usmap, which is needed to make the map plotting function work. I wrote some conditional statements in the plotting fxn to return an error if the required packages are not installed... so they're suggested for the package, but required if the user wants to make the mapping plot.

Reply:
Yeah @sokole, `rgdal` (I was confused by the UI too). If imported/suggested by `usmap` then we don't have to list it in our DESCRIPTION. So remove it unless an `rgdal` func is explicitly used in the PR.

Reply:
okay, I'll remove the packages that are dependencies in `usmap`

Reply:
@clnsmth I had to keep these packages as "suggests" because the vignette fails to build when I run checks if I take them out.

Reply:
@sokole If `usmap` is suggested, then `rgdal` should be suggested as well. Are there other packages here that are raising this issue?

Reply:
@clnsmth looks like we can remove `scales` altogether? I guess the method I ended up using for scaling axis is native in `ggplot2` and not part of `scales`. At least, when I remove it and run checks, it passed.